ui/mirage/factories/job-summary.js

```diff
@@ -18,6 +18,7 @@ export default Factory.extend({
           Running: faker.random.number(10),
           Starting: faker.random.number(10),
           Lost: faker.random.number(10),
+          Unknown: faker.random.number(10),
```

Comment:
**question:**  Should this impact all jobs?

In the edge case, we're developing this feature to enable those users to maintain their up-time requirements. That makes me feel like this situation is only regarding `service` jobs which are long-standing jobs compared to `system` and `batch` jobs.

cc:  @DerekStrickland @mikenomitch do I have this to be, correct?

Reply:
System, sysbatch, service and batch jobs can all transition to unknown.

Reply:
Yeah all of them can be unknown.

Not important for the code, but actually in some ways batch is more vital than service. Since batch jobs are "stateful" more often that service, the cost of restarting them can often be higher. So losing connection and coming back gracefully is more important.